Packet++/header/GtpLayer.h

```diff
@@ -47,16 +47,16 @@ namespace pcpp
 		/** A 1-bit value that states whether there is a N-PDU number optional field */
```

Comment:
it seems like there are a lot of changes in this file and it's hard for me to keep track of them.
Could you please explain the changes you've done here?

Reply:
1. qualifier `const` added to several methods
2. leading spaces have been changed to TABs
3. private section moved to the end of `GtpV1Layer` class declaration
I'm not sure that it is right decision. In some classes this section is placed at the end of declaration while in other it is placed at the beginning. I could not find the document/file with coding conventions.

Reply:
Done